hydromt_fiat/fiat.py

```diff
@@ -1286,19 +1286,24 @@ def read_geoms(self):
 
         fns = glob.glob(Path(self.root, "geoms", "*.geojson").as_posix())
         if self.spatial_joins["aggregation_areas"]:
-            fns_aggregation = glob.glob(Path(self.root, "geoms", "aggregation_areas","*.geojson").as_posix())
-            fns.append(fns_aggregation[0])
+            fns_aggregation = glob.glob(Path(self.root, "geoms", "aggregation_areas/*").as_posix())
```

Comment:
will this now work if this is a shapefile wiht multiple files?

Reply:
a shapefile with multiple layers or multiple files in the folder? 
I tested it with 2 files in the aggregation areas and that seemed to work